lib/event-provider/index.spec.js

```diff
@@ -146,6 +147,10 @@ describe('Nahmii Event Provider', () => {
             fakeSocket.emit('new_receipt', receiptJSON);
             expect(onNewReceipt).to.not.have.been.called;
         });
+
+        it('clears all socket listeners', () => {
+            expect(fakeSocket.removeAllListeners).to.have.been.calledWith();
```

Comment:
Should something else than 'calledWith()' be used.

Reply:
typo, should be `calledonce`